pipeline/vars/bm_node_reimage.bash

```diff
@@ -0,0 +1,109 @@
+#!/bin/bash
+
+# This script will use Teuthology framework to reimage
+# given set of nodes to the os version required
+# This script accepts the hostnames of the nodes as arg
+
+# To invoke this script, pass the argument as below
+# => bm_node_reimage.bash node1 node2 node3 os-version:9.0
```

Comment:
IMO, we should support CLI options for `os-version`. I like the idea of leaving the nodes as positional arguments.